google/cloud/apigeeregistry/v1/admin_models.proto

```diff
@@ -25,6 +25,71 @@ option java_multiple_files = true;
 option java_outer_classname = "AdminModelsProto";
 option go_package = "github.com/apigee/registry/rpc;rpc";
 
+// BuildInfo describes a server build.
+message BuildInfo {
+  // Version of Go that produced this binary.
+  string go_version = 1;
+
+  // The main package path.
+  string path = 2;
+
+  // A module used to create the build.
+  message Module {
+    // The module path.
+    string path = 1;
+    // The module version.
+    string version = 2;
+    // The module checksum.
+    string sum = 3;
+    // An optional replacement for the module
+    Module replacement = 4;
+  }
+
+  // The main module.
+  Module main = 3;
+
+  // The dependencies of the build.
+  repeated Module dependencies = 4;
+
+  // A setting (key-value pair) associated with the build.
+  message Setting {
+    // The key used to store the setting.
+    string key = 1;
+    // The value of the setting.
+    string value = 2;
+  }
+
+  // Settings associated with the build.
+  repeated Setting settings = 5;
+}
+
+// Status represents the status of the service.
+message Status {
+  // A string describing the status.
+  string message = 1;
+
+  // Information about the build of the server.
+  BuildInfo buildinfo = 2;
```

Comment:
Shouldn't this be `build_info`? Maybe just `build` is enough detail though.

Reply:
I might have been overindexing on the naming in the Go runtime/debug package. I think both of your suggestions are better names and prefer `build` for brevity.